README.md

```diff
@@ -122,6 +122,35 @@ Please try the following solutions:
    remote: this repository, visit https://github.com/enterprises/microsoftopensource/sso?authorization_request=AEJANEWOPPW6YTNW5TYNW2K7OBDR3A5PN5ZGOYLONF5GC5DJN5XF62LEZYAF32PCVVRXEZLEMVXHI2LBNRPWSZGODVDHWBVPMNZGKZDFNZ2GSYLML52HS4DFVNHWC5LUNBAWGY3FONZQ
    ```
 
+1. If you see the below errors while cloning the templates repository, this might be caused by the latest `Git Credential Manager Core` has been recently installed and you need to re-authorize the application.
```

Comment:
this is caused by that you used the `Git Credential Manager Core` before GitHub enabled the SSO, and you need to re-authorize the application.